app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express")
+const app = express()
+const {getTopics} = require("./controllers/topics-controllers")
+
+
+app.get("/api/topics", getTopics)
+
+app.all("/api/*", (req, res, next) => {
+    res.status(404).send({msg: "Not Found - path does not exist"})
+})
+
+app.use((err, req, res, next) => {
+    console.log(err)
+    res.status(500).send({msg: "Internal Server Error"})
+})
```

Comment:
This is fantastic! Having this `500` in place will help a lot if you run into bugs later 👍